### PR TITLE
Remove outdated Maia and Hermes info

### PR DESCRIPTION
**Pull request by 0xLightt**
Continuation from: #12574

Context:

Maia's original info should not be displayed. It's tokens were migrated.

Also, Hermes V1 staking data is also not valid atm, since the tokens have been migrated. The only thing  that can be considered live atm from Hermes V1 is the AMM, but it is paused and deprecated, so there is only TVL there because people didn't withdraw it yet.

Another thing to take into account, but not relevant to this PR. Due to swaps being disabled on Hermes V1's AMM, it shouldn't be a price feed.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/hermes-protocol exports TVL: 
        
 
 
    metis                     1.22 M
    
    total                    1.22 M

**Comment by llamatester** [comment]
The adapter at projects/hermes-protocol exports TVL: 
        
 
 
    metis                     1.22 M
    
    total                    1.22 M

**Comment by llamatester** [comment]
The adapter at projects/maia-dao exports TVL: 
        
 
 
    arbitrum-staking          739.84 k
    staking                   739.84 k
    arbitrum                  0
    
    total                    0

File: projects/hermes-protocol/index.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 const {uniTvlExport} = require('../helper/calculateUniTvl.js')
-const { staking } = require('../helper/staking')
 
 module.exports = {
   misrepresentedTokens: true,
   metis:{
     tvl: uniTvlExport("0x633a093C9e94f64500FC8fCBB48e90dd52F6668F", "metis", undefined, undefined, { hasStablePools: true, useDefaultCoreAssets: true, }),
-    staking: staking("0xa4C546c8F3ca15aa537D2ac3f62EE808d915B65b", "0xb27bbeaaca2c00d6258c3118bab6b5b6975161c8"),
```
Comment (g1nt0ki) [inline projects/hermes-protocol/index.js:8]: can you replace this with `staking: () => ({})`

Comment (0xLightt) [reply projects/hermes-protocol/index.js:8]: Done!

**Comment by llamatester** [comment]
The adapter at projects/hermes-protocol exports TVL: 
        
 
 
    metis                     1.27 M
    metis-staking             0
    staking                   0
    
    total                    1.27 M

**Comment by llamatester** [comment]
The adapter at projects/maia-dao exports TVL: 
        
 
 
    arbitrum-staking          757.67 k
    staking                   757.67 k
    arbitrum                  0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/hermes-protocol exports TVL: 
        
 
 
    metis                     1.29 M
    metis-staking             0
    staking                   0
    
    total                    1.29 M

**Comment by llamatester** [comment]
The adapter at projects/maia-dao exports TVL: 
        
 
 
    arbitrum-staking          754.59 k
    staking                   754.59 k
    metis                     0
    metis-staking             0
    arbitrum                  0
    
    total                    0

**Pull request closed without merging**